CONTRIBUTING.md

```diff
@@ -62,3 +62,16 @@ These are recommendations for contributors:
 
   - Include tests for any new or changed functionality
   - Use type annotations liberally
+
+## Adding Changelog Fragments
+
+Any change to the codebase must either include a changelog fragment (in some
+projects these are called "newsfiles") or be in a PR labeled as
```

Comment:
It took me a minute to understand what kind of label is meant here. Could you clarify with something like:
```
Any change to the codebase must either include a changelog fragment (in some
projects these are called "newsfiles") or the PR must have the `no-news-is-good-news`
label attached on github.
```

Reply:
Yeah, the clarification, and need for it, make sense. I'll make the relevant change tomorrow.